.idea/compiler.xml

```diff
@@ -3,6 +3,27 @@
   <component name="CompilerConfiguration">
     <annotationProcessing>
       <profile default="true" name="Default" enabled="true" />
+      <profile name="Gradle Imported" enabled="true">
```

Comment:
What the?  How did this get here

Reply:
Yeah these files get modified by intelliJ. You can tell git to care about them. `git update-index --no-assume-unchanged compiler.xml`